Monika After Story/game/script-farewells.rpy

```diff
@@ -491,6 +491,7 @@ init 5 python:
     )
 
 label bye_prompt_to_work:
+    $ persistent._mas_pm_has_work = None
```

Comment:
`_mas_pm` implies player model, however this isn't being used as a player model variable. Additionally, treating it like a player model solely based around using the `I'm going to work` farewell doesn't make too much sense:
1. Players may use `I'm going to take you somewhere` and bring her to work without telling her they're bringing her to work
2. There's no way to tell her you're no longer working

Additionally, why are you setting this to `None` here only to re-assign it to `True` later? You don't path dialogue around it so I'm not sure the point of not just setting this `True` here.

Also, `persistent` variables should be `default`ed to a value rather than just assigned.